Admin/Get-SimpleAuditLogReport.ps1

```diff
@@ -193,10 +191,7 @@ Process {
     $Result | Add-Member -MemberType NoteProperty -Value $NewValue -Name NewValue
 
     # Add the object to the array to be returned
-    $ResultSet = $ResultSet + $Result
+    $Result
 }
 # Final steps
-End {
-    # Return the array set
-    Return $ResultSet
-}
+End {}
```

Comment:
should remove this if it is empty.

Reply:
<shrug> was left there for consistency with the Begin / Process / End methodology but doesn't matter either way so took it out.

Reply:
True, but there wasn't even a Process section in the script. Which i am surprised that we don't get a rule error on this being that we set the one parameter to be accepted from the pipeline.